source/dlangbot/app.d

```diff
@@ -193,7 +193,12 @@ void handlePR(string action, PullRequest pr)
     auto refs = getIssueRefs(commits);
 
     auto descs = getDescriptions(refs);
-    updateGithubComment(action, refs, descs, pr.commentsURL);
+    auto comment = pr.getBotComment;
+
+    pr.updateGithubComment(comment, action, refs, descs);
+
+    if (refs.length > 0 && comment.body_.length == 0)
+        pr.addLabels(["Bug fix"]);
```

Comment:
In theory there are two other things we might want to consider

- try to read the label to existing PRs (e.g. if the Bot failed to do the first time & for currently open PRs)
- remove the Bugfix label if all issue references has been removed from a PR.

It would be easy to add these two cases as well, but imho they aren't that important and we could save the "additional API requests impact" of this PR.